content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +83,95 @@ and has not been altered during transport.
-   strongly advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails
-   will never reach their recipient.
+   Yahoo or AOL are examples of email providers with a DMARC policy set to `p=reject`. We strongly
+   advise against using an *@yahoo.com* or *@aol.com* address for your users. These emails will
+   never reach their recipient.
 
-``p=none`` is used for the domain owner to receive reports about entities using their domain. It 
+`p=none` is used for the domain owner to receive reports about entities using their domain. It
 should not impact the deliverability if the DMARC check fails.
 
-You can check the DMARC record of a domain name with a tool like
-`MXToolbox DMARC <https://mxtoolbox.com/DMARC.aspx>`_.
+You can check the DMARC record of a domain name with a tool like `MXToolbox DMARC <https://mxtoolbo
+x.com/DMARC.aspx>`_.
 
 If one of your partners, customer or vendor, uses DMARC and has defined one of these policies, the
 Odoo server cannot relay emails from this partner to your users.
 
-You need to :ref:`handle user notifications in Odoo <discuss_app/notification_preferences>`, or replace the 
-email address of the partner with a default email address.
+You need to :ref:`handle user notifications in Odoo <discuss_app/notification_preferences>`, or
+replace the email address of the partner with a default email address.
 
 .. _email_communication/SPFDKIM_common_providers:
 
-SPF, DKIM & DMARC documentation of common providers 
+SPF, DKIM & DMARC documentation of common providers
 ===================================================
 
 - `OVH DNS <https://docs.ovh.com/us/en/domains/web_hosting_how_to_edit_my_dns_zone/>`_
 - `OVH SPF <https://docs.ovh.com/us/en/domains/web_hosting_the_spf_record/>`_
 - `GoDaddy TXT record <https://www.godaddy.com/help/add-a-txt-record-19232>`_
 - `GoDaddy SPF <https://www.godaddy.com/help/add-an-spf-record-19218>`_
 - `GoDaddy DKIM <https://www.godaddy.com/help/add-a-cname-record-19236>`_
-- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-txtspfdkimdmarc-records-for-my-domain/>`_
+- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-
+  txtspfdkimdmarc-records-for-my-domain/>`_
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
 - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
 - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
 
-To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give 
-you a full overview of the content and configuration you have in one email sent! Mail-Tester can 
+To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give
+you a full overview of the content and configuration you have in one email sent! Mail-Tester can
```

Comment:
- Replace `your configuration` with `the configuration`
- Add `use` before `the tool`
- Move the word `tool` after `` `Mail-Tester <https://www.mail-tester.com/>`_ ``
- Replace `will give you` with `, which gives`
- Replace `you have in one email sent` with `in one sent email`
- Replace the exclamation point with a period